src/MountedTraversal.js

```diff
@@ -207,8 +214,14 @@ export function buildInstPredicate(selector) {
       }
       break;
 
+    case 'object':
+      if (!isArray(selector) && !isNull(selector)) {
```

Comment:
do we really need to use an `isNull` or `isArray` function? `Array.isArray` is built into the language, and `=== null` both avoids function call overhead and is equally clear if not more so.